modules/developers-guide/examples/javascript/node-processing1.js

```diff
@@ -0,0 +1,15 @@
+// Get the results from the execute query statement
+// and separate into an array to print out the results
+const resultSet = response.getResultSet();
+const rows = resultSet.getRowsList();
+
+// This for loop gets 2 results
+for ( let i = 0; i < 2; i++) {
```

Comment:
See note on the full sample script - this works, but we could use a `forEach()` and template strings to make this a bit more idiomatic JS.

Reply:
I'll switch it when you modify the grpc-node-client and test it.